WrightTools/data/colors.py

```diff
@@ -0,0 +1,340 @@
+"""COLORS."""
+
+
+# --- import --------------------------------------------------------------------------------------
+
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import collections
+
+import numpy as np
+
+from scipy.interpolate import griddata
+
+from .main import Axis, Channel, Data
+from .. import kit as wt_kit
+
+
+# --- define --------------------------------------------------------------------------------------
+
+
+__all__ = ['from_COLORS']
+
+
+# --- helpers -------------------------------------------------------------------------------------
+
+
+def discover_dimensions(arr, dimension_cols, verbose=True):
```

Comment:
Perhaps these should be _discover_dimensions, otherwise a) this will be a user facing method that is really intended for internal use only, and b) There will be namespace issues with having two copies of this which are both imported in __init__

Reply:
note that `discover_dimensions` is not in `__all__`, so no need to worry about namespace issues as far as I know